x-pack/plugins/security_solution/cypress/integration/fields_browser.spec.ts

```diff
@@ -151,7 +153,7 @@ describe('Fields Browser', () => {
       cy.get(FIELDS_BROWSER_HOST_GEO_CITY_NAME_HEADER).should('exist');
     });
 
-    it('adds a field to the timeline when the user drags and drops a field', () => {
+    it.skip('adds a field to the timeline when the user drags and drops a field', () => {
```

Comment:
Is this one legitimately broken? Any notes on why it was skipped?

Reply:
Before the whole suite was skipped, here I'm skipping just the test that looks like legitimately broken.